### ISSUE TITLE
This does not work

### ISSUE THREAD

**Issue by rebop** [xiaoxiaoflood/firefox-scripts#132]
First, I download Utils_extensions_only.zip. I extract and it creates a folder named utils_extensions_only and inside that a folder named Utils. You do not specify whether to put the Utils FOLDER into the chrome folder in my profile OR the contents of the Utils folder. I tried both. I even tried putting the extracted utils_extensions_only unzipped folder in chrome. And after that clearing the startup cache each time and restating Firefox.

I already had (have) a chrome folder with lots of things I use.

I then tried dragging the xpi to Extensions and also installing extension from file. BOTH show the xpi as being corrupt.

What is wrong?

Thanks.

~Bob

**Comment by xiaoxiaoflood** [comment]
> it creates a folder named utils_extensions_only

It was not instructed to create that folder. You can see the process being executed in the video linked in the instructions. The first screenshot in main page also hints about folders structure in `chrome` folder.

Final structure in your profile folder must be

- chrome (folder)
  - utils (folder)
    - userChrome.jsm
    - ...
  - [anyscript].uc.js
  - ...
 
Not forgetting about files that must be placed in Firefox installation directory.

**Comment by rebop** [comment]
Thanks, but still confused. i have done just that and it does not work. Clainms the xpi is corrupted. and yes, I have cleared the startup cache.

![image](https://user-images.githubusercontent.com/13785042/163883650-e1aa2fe8-ceba-43ab-b8d0-9ca1589a4b84.png)

This is what I have. You can see the path.

I do not understand what you mean here: "Not forgetting about files that must be placed in Firefox installation directory."

**Comment by rebop** [comment]
Ahhh, I see I missed a step. Let me try again. Right back.

**Comment by rebop** [comment]
Got it to work. I had missed a step. and the program files needed to go in the App/Firefox64 folder. Tricky. Now I can test the extension.

Thanks.
